doc/api/fs.md

```diff
@@ -3780,6 +3801,11 @@ returned.
 
 The `FileHandle` has to support reading.
 
+**Note:** If one or more `filehandle.read()` calls are made on a file handle
```

Comment:
We delibirately don't use `**Note:**` anymore afaik?
/cc @nodejs/documentation

Reply:
Makes sense. I removed the Note now.